widget_driver/README.md

```diff
@@ -171,7 +177,7 @@ Now we can continue to the next step!
 ### 2: Code generation
 
 Now it is time to generate some code so that we get our `TestDriver` and `WidgetDriverProvider` set up for us.  
-*If you prefer to do this the old fashioned way without code generation, then that is also possible. 
+*If you prefer to do this the old fashioned way without code generation, then that is also possible.  
```

Comment:
```suggestion
*If you prefer to do this the old fashioned way without code generation, then that is also possible.
```

Reply:
Adding 2 spaces just makes a bit nicer formatting.. 2 spaces forces a new line. 
If you remove the 2 spaces and do a new line in code, then that new line is ignored by the markdown readers..